tests/e2e-jest/config/jest.config.js

```diff
@@ -35,6 +35,9 @@ module.exports = {
 		// Ignore all the files that have utils in the name
 		'utils',
 	],
-	transformIgnorePatterns: [ 'node_modules/(?!(woocommerce)/)' ],
+	transformIgnorePatterns: [
+		'node_modules/?!(simple-html-tokenizer|is-plain-obj|is-plain-object|memize)',
+		'node_modules/@woocommerce/e2e-utils/node_modules/config',
```

Comment:
This is for an exception I ran into running e2e: https://github.com/node-config/node-config/issues/628